src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
+    private int number;
+
+    public void setColour(String colour) {
+        this.colour = colour;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
```

Comment:
The toString() method is well implemented, but make sure to handle potential null values for 'colour' to avoid a NullPointerException. Consider adding null checks or default values.